command/command.go

```diff
@@ -23,6 +24,11 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+func other() {
```

Comment:
The 'other' function does not seem to have any purpose in this code and should be removed.